### PR TITLE
release: prepare for 0.1.7 release

### PR DESCRIPTION
**Pull request by jost-s**
### Summary
~~This branch is based on `develop-0.1` and merges `main-0.1` back into it because I cancelled the release process that would have done that.~~ `main-0.1` was merged into `develop-0.1` separately to avoid squash merge issues.

Secondly I'm adding a one-off semver_increment_mode `patch` to all frontmatters.

A [dry-run based on this branch has been successful](https://github.com/holochain/holochain/actions/runs/6409977107/job/17402305096).

### TODO:
- [ ] CHANGELOG(s) updated with appropriate info
- [ ] Just before pressing the merge button, ensure new entries to CHANGELOG(s) are still under the _UNRELEASED_ heading

### CODE REVIEW

File: crates/release-automation/src/lib/crate_selection/mod.rs [hunk 1]
```diff
@@ -99,7 +99,8 @@ impl<'a> Crate<'a> {
         release_version: &semver::Version,
     ) -> Fallible<Vec<&'a Crate<'a>>> {
         debug!(
-            "setting version to {} in manifest at {:?}",
+            "[{}] setting version to {} in manifest at {:?}",
+            self.name(),
```
Comment (jost-s) [inline crates/release-automation/src/lib/crate_selection/mod.rs:103]: @steveej Are you fine with outputting the package name in the debug logs?

Comment (steveej) [reply crates/release-automation/src/lib/crate_selection/mod.rs:103]: yes and it's already over here :-) https://github.com/holochain/holochain/pull/2896#discussion_r1346350597

Comment (steveej) [reply crates/release-automation/src/lib/crate_selection/mod.rs:103]: this is not a duplicate as i thought it'd be. change is fine with me

**Review by steveej — COMMENTED** [review]
it'll need changing at https://github.com/holochain/holochain/blob/a7c64ba0d3a333da3f0b7a3c382aa0d34fb54771/nix/modules/scripts.nix#L87 to allow patch-level bumps

**Comment by jost-s** [comment]
> it'll need changing at
> 
> https://github.com/holochain/holochain/blob/a7c64ba0d3a333da3f0b7a3c382aa0d34fb54771/nix/modules/scripts.nix#L87
> 
> to allow patch-level bumps

@steveej That section with the flag is not in `scripts.nix` on this branch. Is that used here already?

**Comment by steveej** [comment]
> > it'll need changing at
> > https://github.com/holochain/holochain/blob/a7c64ba0d3a333da3f0b7a3c382aa0d34fb54771/nix/modules/scripts.nix#L87
> > 
> > to allow patch-level bumps
> 
> @steveej That section with the flag is not in `scripts.nix` on this branch. Is that used here already?

maybe a caching issue in your browser. if in doubt check out locally but here it is online: https://github.com/holochain/holochain/blob/develop-0.1/nix/modules/scripts.nix#L87

**Pull request merged**